net/portmapper/portmapper.go

```diff
@@ -724,3 +729,13 @@ func parsePCPResponse(b []byte) (res pcpResponse, ok bool) {
 }
 
 var pmpReqExternalAddrPacket = []byte{0, 0} // version 0, opcode 0 = "Public address request"
+
+const (
+	upnpPort = 1900
```

Comment:
I'm not sure if this will properly detect UPnP on my router, but I'll check it when this gets merged

Reply:
This is only the UDP discovery port number, which is a fixed number. I added a comment.